assets/component-slider.css

```diff
@@ -24,13 +28,20 @@ slider-component {
     scroll-padding-left: 1.5rem;
     -webkit-overflow-scrolling: touch;
     margin-bottom: 1rem;
-    padding-top: max(0.5rem, calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius)));
-    padding-bottom: max(0.5rem, calc(var(--shadow-vertical-offset) + var(--shadow-blur-radius)));
   }
 
   .slider.slider--mobile .slider__slide {
     margin-bottom: 0;
-    padding-bottom: 0;
+    padding-top: max(var(--focus-outline-padding), var(--shadow-padding-top));
+    padding-bottom: max(var(--focus-outline-padding), var(--shadow-padding-bottom));
```

Comment:
Should we also have a padding-top/bottom value when it's not a slider, either on the `ul` or `li:last-child`/`li:first-child`

![](https://screenshot.click/31-26-njj50-lrqwc.png)

Reply:
Curious to learn the settings combination as I wasn't able to get it on [Collection cards](https://screenshot.click/22-01-2uhb0-yqpp6.png), but mainly on [Multicolumn](https://screenshot.click/22-01-bjbwa-zfd8n.png).

Reply:
I just pushed a change for that. I don't want to go down a rabbit hole in this PR, but if that looks better you, we'll roll with it.